TombEngine/Scripting/Internal/TEN/Objects/Moveable/MoveableObject.cpp

```diff
@@ -601,6 +605,22 @@ void Moveable::SetRot(Rotation const& rot)
 	m_item->Pose.Orientation.z = ANGLE(rot.z);
 }
 
+/// Get the moveable's airborne status
+// @function Moveable:GetAirborne
+// @treturn (bool) true if entity animation is flying.
+bool Moveable::GetAirborne() const
+{
+	return m_item->Animation.IsAirborne;
+}
+
+/// Set the moveable's airborne status
+// @function Moveable:SetAirborne
+// @tparam (bool) airborne status for the current animation.
```

Comment:
Same as above, I feel like this would be better worded as: "@return (bool) true if the entity is grounded" or "@return (bool) true if the entity is not in air"

but not sure, I would consult @Sezzary with English stuff :D

Reply:
I may have not breaken my head a lot with this one...

```
/// Set the moveable's airborne status
// @function Moveable:SetAirborne
// @tparam (bool) New airborn status for Lara.
```